models/src_pinterest_ads.yml

```diff
@@ -1,7 +1,7 @@
 version: 2
 
 sources:
-  - name: pinterest_ads
+  - name: pinterest_ads # This source will only be used if you are using a single microsoft_ads source connector. If multiple sources are being unioned, their tables will be directly referenced via adapter.get_relation.
```

Comment:
Similar to Linkedin, can we update this package to ensure we do not need to make an update to fivetran_utils

Reply:
Yes. I updated the identifiers to follow the expected format. Update to fivetran_utils no longer needed.